src/inputs.ts

```diff
@@ -17,12 +41,51 @@ const parseJSON = <T>(getInput: GetInput, property: string): T | undefined => {
   }
 };
 
+function reverseString(input_string: string): string {
+  return input_string.split('').reverse().join('');
```

Comment:
Semi-irrelevant to utf-8 but this will not properly reverse some characters with surrogate pairs and it kind of promises that it will work on any string.

```suggestion
  return [...input_string].reverse().join('');
```

This change should deal with that problem better.

I'd also like to comment that we're mixing case and camel case in our TS. Camel case is the typical standard.